astropy/io/votable/tree.py

```diff
@@ -2569,18 +2588,32 @@ def is_empty(self):
         """
         return self._empty
 
-    def create_arrays(self, nrows=0, config=None):
+    def create_arrays(
+        self,
+        nrows=0,
+        config=None,
+        *,
+        colnumbers=None,
```

Comment:
Does this count as API change? If so, should mention this new keyword in the change log too?

Reply:
You're right. It also made me realize that the docstring is slightly wrong. Thanks !

Reply:
done !